src/query/api/v1/handler/prometheus/handleroptions/headers.go

```diff
@@ -49,6 +49,19 @@ const (
 	// Valid values are "unaggregated" or "aggregated".
 	MetricsTypeHeader = M3HeaderPrefix + "Metrics-Type"
 
+	// WriteTypeHeader sets whether to perform a default write that can be
+	// written to unaggreated namespace and optionally aggregated or
+	// to specifically only perform aggregations (and skip writing unaggregated
+	// values).
+	// Valid values are "default" or "aggregate"
```

Comment:
Maybe a bit clearer something like this? Up to you, either way `unaggreated`->`unaggregated`

```
	// WriteTypeHeader is a header that controls if default
        // writes should be written to both unaggregated and aggregated
	// namespaces, or if unaggregated values are skipped and 
        // only aggregated values are written. 
	// Valid values are "default" or "aggregate"
```

Reply:
Sure thing, sounds better 👍